sources/packages/web/src/views/institution/locations/active-applications/ActiveApplicationSummaryData.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <body-header
+    title="Applications"
+    :recordsCount="applications.results?.length"
+    class="m-1"
+  >
+    <template #actions>
+      <div class="search-box-display-width">
+        <v-text-field
+          label="Search name or application #"
+          prepend-inner-icon="mdi-magnify"
+          variant="outlined"
+          v-model="searchCriteria"
+          @keyup.enter="searchActiveApplications"
+        ></v-text-field>
+      </div>
+    </template>
+  </body-header>
+  <br />
+  <content-group>
+    <DataTable
+      :value="applications.results"
+      :lazy="true"
+      class="p-m-4"
+      :paginator="true"
+      :rows="pageLimit"
+      :rowsPerPageOptions="rowsPerPageOptions"
+      :totalRecords="applications.count"
+      @page="pageEvent"
+      @sort="sortEvent"
+    >
+      <template #empty>
+        <p class="text-center font-weight-bold">No records found.</p>
+      </template>
+      <Column
+        field="fullName"
+        header="Name"
+        :sortable="true"
+        headerStyle="width: 20%"
+      >
+        <template #body="slotProps">
+          <span>{{ slotProps.data.fullName }}</span>
```

Comment:
I believe that we can avoid creating a template when the value can be directly assigned to the column.